src/Provider/LocationIQ/LocationIQ.php

```diff
@@ -32,7 +33,15 @@ final class LocationIQ extends AbstractHttpProvider implements Provider
     /**
      * @var string
      */
-    const BASE_API_URL = 'https://locationiq.org/v1';
+    public $baseUrl = 'https://{region}.locationiq.com/v1';
```

Comment:
I don't think this needs to be `public` ; I would make it `private`.

Reply:
I can make it protected if you want but the reason I made it public was because the constant it replaces was public and there may be users that are accessing it for some reason - which I didn't want to prevent.

Reply:
```suggestion
    const BASE_API_URL = 'https://{region}.locationiq.com/v1';
```

You're right!
Removing the `BASE_API_URL` would create a breaking change anyway which is something we want to avoid.

I would keep the `BASE_API_URL` and use that constant to populate `$this->baseUrl` (and make that `baseUrl` property `protected` so there is no breaking change.

Constants are (still) `public` in our code because the visibility on class constants only came with PHP 7.1

Reply:
I've reinstated the BASE_API_URL constant and used that to populate the now protected $baseUrl as requested.